code/index.html

```diff
@@ -1,30 +1,118 @@
+  <link href="https://fonts.googleapis.com/css2?family=Permanent+Marker&display=swap" rel="stylesheet">
+</head>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+<body>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+  <!-- HERO IMAGES -->
+
+  <header class="hero">
+    <picture>
+      <source srcset="./images/diving-mobil.jpg" media="(max-width: 767px)">
+      <source srcset="./images/diving-tablet.jpg" media="(max-width: 1024px)">
+      <img class="hero-image" src="./images/diving-desktop.jpg" alt="Hero Image">
+    </picture>
+    <h1 class="hero-title">Ocean clean up</h1>
+    <p class="p-header">Our mission is to keep the amazing oceans clean and healthy for all creatures to enjoy for
+      generations to come. Join us on
+      a clean up dive!</p>
+  </header>
+
+  <div class="h2">
+    <h2>To the rescue!</h2>
+  </div>
+
+  <!-- ACCORDION -->
+
+  <section class="big-container">
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <section class="accordion">
+      <button class="acc-button" id="button1">What we do</button>
+      <div class="info">
+        <p class="info-content" id="info-content-1">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+      </div>
+      <button class="acc-button" id="button2">How you can help</button>
+      <div class="info">
+        <p class="info-content" id="info-content-2">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
       </div>
     </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+
+    <!-- SIGNUP FORM -->
+    <section class="form-container">
+
+      <form action="http://httpbin.org/anything" method="POST">
+
+        <div class="row">
+          <div class="colum25">
+            <label for="fullname">Full Name</label>
+          </div>
+          <div class="colum75">
+            <input class="text-field" type="text" name="fullname" required>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="colum25">
+            <label for="email">E-mail</label>
+          </div>
+          <div class="colum75">
+            <input class="text-field" type="text" name="email" id="email" required>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="colum25">
+            <label for="logged-dives">Logged Dives</label>
+          </div>
+          <div class="colum75">
+            <select name="logged-dives" id="Logged Dives" required>
+              <option value="0-300">0-300</option>
+              <option value="300-600">300-600</option>
+              <option value="600+">600+</option>
+            </select>
```

Comment:
Nice with a dropdown! There are many cool form elements we can use for number inputs, depending on how we want to split our data, and sometimes it's hard to choose the right one. I was thinking of suggesting a range slider (because I think they are cool and intuitive when it comes to number inputs) but when you only want three options like this, a dropdown works really good!